katas/content/qubit/index.md

```diff
@@ -115,8 +115,8 @@ $$|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$$
 Several ket symbols have a generally accepted use, such as:
 
 <table>
-    <col width=180>
-    <col width=180>
+    <col>
+    <col>
```

Comment:
I believe this `<col>` tags are not longer needed after removing the `width` attribute.